interpreter/llvm/src/tools/clang/lib/CodeGen/CGException.cpp

```diff
@@ -533,6 +533,14 @@ void CodeGenFunction::EmitEndEHSpec(const Decl *D) {
     return;
 
   ExceptionSpecificationType EST = Proto->getExceptionSpecType();
+
+  // Might need to deserialize
+  if (EST == EST_Uninstantiated || EST == EST_Unevaluated) {
```

Comment:
I suspect we can fix this in the ASTReader: https://github.com/root-project/root/blob/master/interpreter/llvm/src/tools/clang/lib/Serialization/ASTReader.cpp#L10155-L10196

We should detect this case when deserializing and complete the redecl chain there.